### PR TITLE
Added example to set_tbl_cols

### PR DESCRIPTION
**Pull request by keiv-fly**
Want to add example to set_tbl_cols because I could not use it properly. Maybe a silly mistake from my side, but I could not find the answer in the documentation with a first glance.

### CODE REVIEW

File: py-polars/polars/cfg.py [hunk 1]
```diff
@@ -64,6 +64,14 @@ def set_tbl_cols(cls, n: int) -> "Type[Config]":
         ----------
         n
             number of columns to print
+        
+        Examples
+        --------
+>>> import polars as pl
```
Comment (ritchie46) [inline py-polars/polars/cfg.py:70]: Could you indent this to the same level as the documentation? And maybe remove the latest two rows as they don't do anything?

**Comment by keiv-fly** [comment]
Is it better now? I deleted the last row, but did not understand why should it be two rows. "df" shows the dataframe. And `"""` was on the new line.

File: py-polars/polars/cfg.py [hunk 2]
```diff
@@ -64,6 +64,14 @@ def set_tbl_cols(cls, n: int) -> "Type[Config]":
         ----------
         n
             number of columns to print
+        
+        Examples
+        --------
+
+        >>> import polars as pl
```
Comment (zundertj) [inline py-polars/polars/cfg.py:71]: In the docstring examples, we assume `import polars as pl` has already been done, so this line can be removed.

File: py-polars/polars/cfg.py [hunk 3]
```diff
@@ -64,6 +64,14 @@ def set_tbl_cols(cls, n: int) -> "Type[Config]":
...
+        >>> pl.cfg.Config.set_tbl_cols(1000)
+        >>> df = pl.DataFrame({str(i):[i] for i in range(100)})
+        >>> df
```
Comment (ritchie46) [inline py-polars/polars/cfg.py:74]: In other examples we always show the result. But given that there are 100 columns it is too much to show, so maybe take a smaller number of columns and show the resulting table.

File: py-polars/polars/cfg.py [hunk 4]
```diff
@@ -64,6 +64,21 @@ def set_tbl_cols(cls, n: int) -> "Type[Config]":
...
+        shape: (1, 100)
+        ┌─────┬─────┬─────┬─────┬─────┬─────┐
+        │ 0   ┆ 1   ┆ 2   ┆ ... ┆ 98  ┆ 99  │
+        │ --- ┆ --- ┆ --- ┆     ┆ --- ┆ --- │
+        │ i64 ┆ i64 ┆ i64 ┆     ┆ i64 ┆ i64 │
+        ╞═════╪═════╪═════╪═════╪═════╪═════╡
+        │ 0   ┆ 1   ┆ 2   ┆ ... ┆ 98  ┆ 99  │
+        └─────┴─────┴─────┴─────┴─────┴─────┘
```
Comment (keiv-fly) [inline py-polars/polars/cfg.py:81]: I changed as advised

Comment (ritchie46) [reply py-polars/polars/cfg.py:81]: Thanks!

**Comment by keiv-fly** [comment]
I changed so that it is black and blackdoc formatted.

**Comment by ritchie46** [comment]
Thanks @keiv-fly. Merging it in. :+1:

**Pull request merged**